tests/unit/test_umbral_keys.py

```diff
@@ -133,6 +119,18 @@ def test_key_encoder_decoder(random_ec_curvebn1):
     assert decoded_key.to_bytes() == umbral_key.to_bytes()
 
 
+def test_public_key_as_hex(random_ec_curvebn1):
+    pubkey = UmbralPrivateKey(random_ec_curvebn1, default_params()).get_pubkey()
+    hex_string = pubkey.hex()
```

Comment:
Maybe also can add a check that it's indeed hex and nothing was messed up, such as:
```
assert set(hex_string) == set('0123456789abcdef')
assert len(hex_string) == 66
```

Reply:
✅